_includes/footer.html

```diff
@@ -2,18 +2,20 @@
   <div class="contain">
     <img
       alt=""
+      role="presentation"
+      aria-hidden="true"
       class="marg-b-2"
       height="40"
       src="/assets/img/logo.svg"
       width="40"
     />
-    <ul class="d:flex flex-wrap d:flex-row justify-center list-style-none">
+    <ul class="d:flex flex-wrap d:flex-row list-style-none">
       {% for link in site.links %}
         <li class="d:c-1/3 marg-b-3 d:marg-r-3">
           {% if link.newTab %}
             <a
-              nofollow
-              noreferrer
+              class="black-important"
+              rel="nofollow noopener noreferrer"
```

Comment:
i was too used to react and did this wrong before - this is how it's actually supposed to look